app/models/curriculum.rb

```diff
@@ -1,4 +1,5 @@
 class Curriculum < ApplicationRecord
+  validates :name, presence: true
```

Comment:
Additionally, we should force through the migration that the attribute is necessary for the object.
https://edgeguides.rubyonrails.org/active_record_migrations.html#writing-a-migration

Reply:
Agree